.github/workflows/ci.yaml

```diff
@@ -0,0 +1,22 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - "main"
+      - "dev*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
+      - name: Build
+        run: |
+          ./gradlew clean build
+      - name: Unit testing
```

Comment:
I believe `./gradlew build` will also run the unit tests, so we can remove this step

Reply:
sweet, done as of [16e16e9](https://github.com/PPartisan/RockPaperScissorsTests/pull/3/commits/16e16e9f35443a8fc908d4ec27de86a217ef4376)